modules/database/sqlite/load.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Module Name: SQLite Integration
+ * Description: Use an SQLite database instead of MySQL. CAUTION: Enabling this module will bring up the WordPress installation screen. You will need to reconfigure your site, and you will lose all your data. If you then disable the module, you will get back to your previous MySQL database, with all your previous data intact.
```

Comment:
This is still an extremely long module description, which mostly doesn't actually describe the module but is a warning, which is not the purpose of this field. :)

So I would still advise to add this warning as custom HTML inside `admin/load.php` - even if it's a "hack" outside of this module's code, it's still cleaner IMO than misusing the description as a warning.

Reply:
Implemented using a `Notice` header in modules. This way we'll have the implementation available for other modules in case we ever need it, and it's less "hacky" than hardcoding an exception in `admin/load.php` 👍